packages/jsii-dotnet-analyzers/NuGet.Metadata.props.t.js

```diff
@@ -2,17 +2,25 @@ const package = require('./package.json');
 
 process.stdout.write(`<Project>
   <PropertyGroup>
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <IncludeSymbols>True</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <IncludeSource>True</IncludeSource>
-    <PackageOutputPath>..\\..\\bin\\$(Configuration)\\NuGet\\</PackageOutputPath>
     <PackageVersion>$(JsiiVersion)</PackageVersion>
     <Description>${package.description}</Description>
-    <ProjectUrl>${package.homepage}</ProjectUrl>
+    <PackageLicenseExpression>${package.license}</PackageLicenseExpression>
     <LicenseUrl>https://spdx.org/licenses/${package.license}.html</LicenseUrl>
```

Comment:
LicenseUrl is deprecated. It can be removed now that we are using PackageLicenseExpression.

Same with the other files in this PR.